### ISSUE TITLE
Chart line rounding doesn't match numbers shown below

### ISSUE THREAD

**Issue by coreycaitlin** [18F/doi-extractives-data#2119]
In charts we have a labelled y-axis line above the chart, which is supposed to help ground the user and help provide a sense of scale. In some cases, this line is situated above the top bar, but the label of the line is less than the highest bar amount.

![screen shot 2016-10-27 at 5 01 22 pm](https://cloud.githubusercontent.com/assets/1450354/19900340/62a3ac2c-a020-11e6-82fd-1ba7a12e4e73.png)

![screen shot 2016-10-27 at 4 59 42 pm](https://cloud.githubusercontent.com/assets/1450354/19900345/65c96cde-a020-11e6-840e-3b62db10e1ff.png)

**Comment by gemfarmer** [comment]
@coreycaitlin Ugh this issue. 

The problem is that it is hard to round AND be accurate. The value that is being rounded is always 5% greater than the highest value in a given bar chart, but, if we are rounding it to 1 significant figure, we can't truly avoid being either over/under that value. 

Maybe one potential solution could be to have it round to 1 significant figure by default, but round to 2 if the rounded value is less than the highest bar chart value?

Example:
```
highest_bar_value: $500
rounded value: $525
rounded_1_sigfig: $500
rounded_2_sigfig: $530

highest_bar_value: $12,140,968
rounded value: $12,748,016.40
rounded_1_sigfig: $10 million
rounded_2_sigfig: $13 million

highest_bar_value: 10,466 jobs
rounded value: 10,989.3 jobs
rounded_1_sigfig: 10k jobs
rounded_2_sigfig: 11k jobs

In instances that it is already working, this wouldn't have to apply:
highest_bar_value: $480
rounded value: $504
rounded_1_sigfig: $500
~~rounded_2_sigfig: $500~~
```

**Comment by coreycaitlin** [comment]
That sounds brilliant to me @gemfarmer! I think slightly-oddly-specific is better than confusing, so no objection to more specific values from me.

**Comment by gemfarmer** [comment]
Closed with #2175
